utils/extract_license_edition.sh

```
@@ -0,0 +1,116 @@
+    printf '%s\n' "$output" >&2
+    return 2
+  fi
+
+  edition=$(printf '%s' "$output" | tr -d '\r' | sed -n '1p')
+  if [ -z "$edition" ]; then
+    return 2
+  fi
+
+  printf '%s' "$edition"
+  return 0
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Delete it.

(SC2112)